packages/@n8n/syslog-client/test/fixtures/certificate.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This certificate is valid till `Dec 16 13:24:01 2026 GMT` does it make sense to make this valid much longer 25 years? Otherwise I would imagine the test will start failing on that date next year.